eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -292,6 +294,33 @@ registerDRep = do
   dreps `shouldSatisfy` Map.member (KeyHashObj khDRep)
   pure khDRep
 
+-- | In contrast to `setupSingleDRep`, this function does not make a UTxO entry
+-- that could count as delegated stake to the DRep, so that we can test that
+-- rewards are also calculated nonetheless.
+setupDRepWithoutStake ::
+  forall era.
+  ( ConwayEraTxCert era
+  , ShelleyEraImp era
+  ) =>
+  ImpTestM
+    era
+    ( KeyHash 'DRepRole (EraCrypto era)
+    , KeyHash 'Staking (EraCrypto era)
+    )
+setupDRepWithoutStake = do
+  drepKH <- registerDRep
+  delegatorKH <- freshKeyHash
+  submitTxAnn_ "Delegate to DRep" $
+    mkBasicTx mkBasicTxBody
+      & bodyTxL . certsTxBodyL
+        .~ SSeq.fromList
+          [ mkRegDepositDelegTxCert @era
+              (KeyHashObj delegatorKH)
+              (DelegVote (DRepCredential $ KeyHashObj drepKH))
+              zero
```

Comment:
We should lookup up the deposit in PParmas similar to `setupPoolWithoutStake`. I'd like to make existing tests a bit more realistic in close future by setting more of the PParams at the beginning of all tests.
```suggestion
              deposit
```